cmd/moroz/main.go

```diff
@@ -56,6 +56,7 @@ func main() {
 		flEvents    = flag.String("event-logfile", envString("MOROZ_EVENTLOG_FILE", "/tmp/santa_events"), "path to file for saving uploaded events")
 		flVersion   = flag.Bool("version", false, "print version information")
 		flHTTPDebug = flag.Bool("http-debug", false, "enable debug for http(dumps full request)")
+		flNoTLS     = flag.Bool("tls-handled-elsewhere", false, "I promise I terminated TLS elsewhere")
```

Comment:
I like the help message, but maybe lets set the flag to `-tls` and just have it `-tls=false` on startup.